### PR TITLE
test: segregate long running integration tests

### PR DESCRIPTION
**Pull request by qued**
|                      | Time |
| --------------------- | ----- |
| Before changes: | 13:15 |
| After changes:  | 10:18 |
| Percent decrease: | 22% |

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 0.10.28-dev0
+
+### Enhancements
+
+* **Separated integration tests to reduce test time** Used `pytest` markers to mark long-running integration tests, to facilitate speedier local testing and to run separately in CI.
```
Comment (cragwolfe) [inline CHANGELOG.md:5]: i'm not sure "integration" is the right adjective here

Comment (cragwolfe) [reply CHANGELOG.md:5]: why not just call it "chipper," like it is?
also makes sense to move chipper tests to their own file IMO

Comment (qued) [reply CHANGELOG.md:5]: Not all the tests I've marked are chipper.

**Comment by qued** [comment]
Closing for now as we are not currently going in this direction.

**Pull request closed without merging**